epsagon/trace.py

```diff
@@ -953,6 +953,15 @@ def remove_ignored_keys(self, input_dict):
                 value = input_dict[key]
                 if isinstance(value, dict):
                     copied_dict[key] = self.remove_ignored_keys(value)
+                else:
+                    try:
+                        json_value = json.loads(value)
+                        if isinstance(json_value, dict):
+                            copied_dict[key] = (json.dumps(
+                                self.remove_ignored_keys(json_value)
+                            ))
+                    except (TypeError, json.errors.JSONDecodeError):
```

Comment:
when can TypeError be raised?

Reply:
Depends on the Python version you'll get either one of these